### DISCUSSION TITLE
How to access all linescans and save to csv

### DISCUSSION THREAD

**Discussion by halley276** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#76] [Q&A]
Hi, I am trying to access all the linescans at once so that I can export it to csv and run it locally. I saw a line of code provided which indexed individual linescans, so I tried to remvoe the 'id' parameter. It didn't work and I received this error, which I don't understand how to fix. 
![Screenshot 2021-06-01 at 10 43 59 PM](https://user-images.githubusercontent.com/65350949/120343496-5ad74700-c32b-11eb-8fdb-c5a842932b5a.png)

I then tried to index individual elements, and use a loop to concatenate them based on the time dimension (I'm not too sure whether this is the right dimension to use). It seemed to work, but when I tried to export to csv, the function is not recognised. I have installed the xarray_extras package and it seemed to be imported successfully. I have read the documentation and do not see where I went wrong. Could someone help me with this please? 
![Screenshot 2021-06-01 at 10 45 43 PM](https://user-images.githubusercontent.com/65350949/120343617-73dff800-c32b-11eb-95ac-1df575ab81c1.png)

**Comment by EY-glowglowglow** [comment]
hi @patrickmuston1 would you know how to fix this issue?
